src/server/routes/classified_reports.ts

```diff
@@ -0,0 +1,63 @@
+import { Worker, MessageChannel } from "node:worker_threads";
+import * as path from "node:path";
+import { Request, Response } from "express";
+import type { Logger } from "winston";
+
+import { endWithStatusAndBody, getParsedUrl } from "../helpers/http";
+
+// Create a separate Worker thread to do the requests on since the
+// data processing is CPU intensive and would otherwise interfere
+// with mainthread responsiveness.
+// For now, only a single worker is used though multiple BQ requests
+// can be in-flight on it.
+const worker = new Worker(path.join(__dirname, "..", "helpers", "classified_reports_transform.ts"));
+
+export default async function handleClassifiedReports(logger: Logger, req: Request, res: Response) {
```

Comment:
I copied this over from `handleUserReports`, but has been wondering if there is better way to handle this to avoid duplication. There is a new parameter `paramPrediction`, but otherwise it's pretty much the same (the worker functions are different though).

Reply:
Ok nvm, I've just created a helper function for this in a separate commit :)